.github/workflows/develop_commit_notification.yml

```diff
@@ -29,5 +29,6 @@ jobs:
             "app_name": "oppiaserver-backup-migration",
```

Comment:
Also just wanted to check why this is specified here? I'd have thought it'd be server-agnostic.

Reply:
Yeah, currently the release process is not server-agnostic